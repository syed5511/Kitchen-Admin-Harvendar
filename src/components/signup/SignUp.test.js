import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import SignUp from './SignUp.component';


describe('SignUp', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<SignUp />).toJSON();
        expect(tree).toMatchSnapshot();
      });

   it('renders without crashing', () => {
      shallow(<SignUp />);
    });
});