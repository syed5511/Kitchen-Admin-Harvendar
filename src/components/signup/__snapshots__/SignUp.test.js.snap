// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp renders correctly 1`] = `
<div
  class="signup-container"
>
  <div
    class="img-container"
  />
  <div
    class="inner-container"
  >
    <form
      action="#"
      class="signup"
      className="av-valid"
      method="get"
      noValidate={true}
      onKeyDown={[Function]}
      onSubmit={[Function]}
    >
      <h3>
        Kitchen Registration
      </h3>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="name"
        >
          Kitchen Name (OR) Company Name
        </label>
        <input
          className="is-untouched is-pristine av-valid form-control"
          id="name"
          maxLength={24}
          minLength={6}
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyUp={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="address"
        >
          Address(Depot)
        </label>
        <input
          className="is-untouched is-pristine av-valid form-control"
          id="address"
          minLength={6}
          name="address"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyUp={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        class="horizontal-container"
      >
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="zip_code"
          >
            Enter Zip Code
          </label>
          <input
            className="is-untouched is-pristine av-valid form-control"
            id="zip_code"
            maxLength={8}
            minLength={6}
            name="zip_code"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onInput={[Function]}
            onKeyUp={[Function]}
            required={true}
            type="number"
            value=""
          />
        </div>
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="state"
          >
            Select State
          </label>
          <select
            className="form-control"
            id="state"
            name="state"
            onChange={[Function]}
          >
            <option
              value="UP"
            >
              STATE-1
            </option>
            <option
              value="UP"
            >
              STATE-2
            </option>
            <option
              value="UP"
            >
              STATE-3
            </option>
            <option
              value="UP"
            >
              STATE-4
            </option>
            <option
              value="UP"
            >
              STATE-5
            </option>
          </select>
        </div>
      </div>
      <div
        class="horizontal-container"
      >
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="city"
          >
            Select City
          </label>
          <select
            className="form-control"
            id="city"
            name="city"
            onChange={[Function]}
          >
            <option
              value="UP"
            >
              Noida
            </option>
            <option
              value="UP"
            >
              Delhi
            </option>
            <option
              value="Delhi"
            >
              Hyderabad
            </option>
            <option
              value="Delhi"
            >
              Pune
            </option>
            <option
              value="Delhi"
            >
              Agra
            </option>
            <option
              value="Delhi"
            >
              Gugaon
            </option>
          </select>
        </div>
        <div
          className="form-group"
        >
          <label
            className=""
            htmlFor="contact"
          >
            Contact
          </label>
          <input
            className="is-untouched is-pristine av-valid form-control"
            id="contact"
            maxLength={10}
            minLength={10}
            name="contact"
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            onInput={[Function]}
            onKeyUp={[Function]}
            required={true}
            type="tel"
            value=""
          />
        </div>
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="email"
        >
          Email Address
        </label>
        <input
          className="is-untouched is-pristine av-valid form-control"
          id="email"
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyUp={[Function]}
          required={true}
          type="email"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="password"
        >
          Enter Password
        </label>
        <input
          className="is-untouched is-pristine av-valid form-control"
          id="password"
          maxLength={16}
          minLength={8}
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyUp={[Function]}
          required={true}
          type="password"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className=""
          htmlFor="re_password"
        >
          Re-Enter Password
        </label>
        <input
          className="is-untouched is-pristine av-valid form-control"
          id="re_password"
          name="re_password"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyUp={[Function]}
          type="password"
          value=""
        />
      </div>
      <div
        className="form-check"
      >
        <input
          checked={false}
          className="is-untouched is-pristine av-valid form-check-input"
          id="checkbox_agree"
          name="checkbox_agree"
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          onInput={[Function]}
          onKeyUp={[Function]}
          required={true}
          type="checkbox"
          value={false}
        />
        <label
          className=""
          htmlFor="checkbox_agree"
        >
          I agree with terms and conditions
        </label>
      </div>
      <button
        aria-label={null}
        className="btn btn-primary"
        onClick={[Function]}
      >
        Sign Up
      </button>
      <div
        class="horizontal-container mb-5"
      >
        <label
          className="btn text-left m-0 p-0"
        >
          Login
        </label>
        <label
          className="btn text-right m-0 p-0"
        >
          Forgot Password
        </label>
      </div>
    </form>
  </div>
</div>
`;
